app/views/posts/index.html.erb

```diff
@@ -1,41 +1,32 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Posts</h1>
+<div class="container d-flex">
+  <div class="row align-self-center"><h2>Posts</h2></div>
+</div>
 
-<table>
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Body</th>
-      <th>Author</th>
-      <th>Replies count</th>
-      <th>Votes count</th>
-      <th>Favorites count</th>
-      <th>Follows count</th>
-      <th>Tags count</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class="container">
+  <% @posts.each do |post| %>
```

Comment:
One of my rules of thumb is whenever I see an `each` loop in a View template that is looping over a list of records, I decided whether or not it makes sense to create a partial for the view logic that exists in the loop. 

Especially if AJAX is being considered, it's a lot easier to do AJAX with a partial. If you think that might be important, or if the index page is getting to be unwieldy, it might make sense to make a partial.

Reply:
Yep. Great advice and it's to be done shortly.